src/managers/builtin/venvUtils.ts

```diff
@@ -347,6 +347,90 @@ export async function getGlobalVenvLocation(): Promise<Uri | undefined> {
     return undefined;
 }
 
+async function createWithCustomization(version: string): Promise<boolean | undefined> {
+    const selection: QuickPickItem | undefined = await showQuickPick(
+        [
+            {
+                label: VenvManagerStrings.quickCreate,
+                description: VenvManagerStrings.quickCreateDescription,
+                detail: l10n.t('Will use Python version {0} and install workspace dependencies .', version),
+            },
+            {
+                label: VenvManagerStrings.customize,
+                description: VenvManagerStrings.customizeDescription,
+            },
+        ],
+        {
+            placeHolder: VenvManagerStrings.selectQuickOrCustomize,
+            ignoreFocusOut: true,
+        },
+    );
+
+    if (selection === undefined) {
+        return undefined;
+    } else if (selection.label === VenvManagerStrings.quickCreate) {
+        return false;
+    }
+    return true;
+}
+
+async function createWithProgress(
+    nativeFinder: NativePythonFinder,
+    api: PythonEnvironmentApi,
+    log: LogOutputChannel,
+    manager: EnvironmentManager,
+    basePython: PythonEnvironment,
+    venvRoot: Uri,
+    envPath: string,
+    packages?: string[],
+) {
+    const pythonPath =
+        os.platform() === 'win32' ? path.join(envPath, 'Scripts', 'python.exe') : path.join(envPath, 'bin', 'python');
+
+    return await withProgress(
+        {
+            location: ProgressLocation.Notification,
+            title: VenvManagerStrings.venvCreating,
+        },
+        async () => {
+            try {
+                const useUv = await isUvInstalled(log);
+                if (basePython.execInfo?.run.executable) {
+                    if (useUv) {
+                        await runUV(
+                            ['venv', '--verbose', '--seed', '--python', basePython.execInfo?.run.executable, envPath],
+                            venvRoot.fsPath,
+                            log,
+                        );
+                    } else {
+                        await runPython(
+                            basePython.execInfo.run.executable,
+                            ['-m', 'venv', envPath],
+                            venvRoot.fsPath,
+                            manager.log,
+                        );
+                    }
+                    if (!(await fsapi.pathExists(pythonPath))) {
+                        log.error('no python executable found in virtual environment');
```

Comment:
Should we hid the quick create option if we already know there is no python? Eventually we could auto install python for the user but in the meantime it might be better to hide

Reply:
This is actually a different thing. This is after creating the venv, we could not find a python in the virtual environment. Typically this should not happen at all. This is just me being overly cautious.

Reply:
We actually throw error the moment user clicks create with no python installed.